Makefile

```diff
@@ -0,0 +1,40 @@
+TERRAFORM_VERSION = 1.2.8
```

Comment:
Can we update this

Reply:
im not sure....

Reply:
All the projects on the shared pipeline specify the same version